audnauseum/gui/ui.py

```diff
@@ -137,11 +137,13 @@ def whichbtn(ui, looper: Looper, _str):
 
     if _str == 'reverse':
         # TODO need function in looper to set reverse
```

Comment:
There is a toggle control in Looper:
def track_toggle_reverse(self, track):
        track.fx.is_reversed = not(track.fx.is_reversed)

Reply:
I was just thinking we disable the button since we didn't have time to implement the feature, but yeah, those would be necessary to connect the UI to the tracks in the future. Maybe in the future it would be a toggle instead of button